Game/Source/GDKTestGyms/BenchmarkGymGameModeBase.cpp

```diff
@@ -0,0 +1,243 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "BenchmarkGymGameModeBase.h"
+
+#include "Engine/World.h"
+#include "EngineClasses/SpatialNetDriver.h"
+#include "GDKTestGymsGameInstance.h"
+#include "Interop/SpatialWorkerFlags.h"
+#include "Misc/CommandLine.h"
+#include "Utils/SpatialMetrics.h"
+
+DEFINE_LOG_CATEGORY(LogBenchmarkGymGameModeBase);
+
+const FString ABenchmarkGymGameModeBase::TotalPlayerWorkerFlag = TEXT("total_players");
```

Comment:
Could put these in the anon namespace if the blueprint functions were implemented in the cpp, although can see why it's done like this too so just a thought

Reply:
Yeah either or. I don't mind personally.